src/db.js

```diff
@@ -1,4 +1,7 @@
+import mongoose from 'mongoose'
+
 export default callback => {
-  // connect to a database if needed, then pass it to `callback`:
-  callback()
+  mongoose.connect('mongodb://localhost/livingDB')
```

Comment:
Make this url configurable. Maybe also add come commandline parsing to set the config.

https://www.npmjs.com/package/commander

Reply:
Created a task for commandline parsing.